_rules/meta-viewport.md

```diff
@@ -0,0 +1,116 @@
+---
+id: 
+name: meta viewport
+rule_type: atomic
+description: |
+  This rule checks that the meta element is not used to block the user agent ability to zoom 
+accessibility_requirements:
+  wcag20:1.4.10: # Reflow (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to the first valid `<meta name="viewport">` element with a `content` attribute in a document.
```

Comment:
why only the first valid one? Does having multiple override the previous meta?

Reply:
@jkodu **I think you're right, and we need to focus every meta viewport in the document.**

I based the writing of the rule on the meta refresh rule (and that is the applicability for a meta refresh). But I now did some tests for the meta viewport :-) 

And I find two different behaviors depending on properties (`user-scalable` or `maximum-scale`)

````html
	<meta name="viewport" content="width=device-width, user-scalable=no" />
	<meta name="viewport" content="width=device-width, user-scalable=yes" />
````
The `user-scalable=yes` is on the last meta:
- FF/Android -> the user can zoom
- CH/Android -> the user can zoom
- Internet/Android - Safari/iOS  -> the user can zoom

````html
	<meta name="viewport" content="width=device-width, user-scalable=yes" />
	<meta name="viewport" content="width=device-width, user-scalable=no" />
````

The `user-scalable=yes` is on the first meta:
- FF/Android -> the user can't zoom
- CH/Android -> the user can't zoom
- Internet/Android - Safari/iOS  -> the user can zoom

So we have cases where the browser take into account the last declaration of the meta viewport for the `user-scalable` property, and cases where browser just don't care about :-)

**BUT** for the `maximum-scale` property, there are differences. 
In some browsers, whatever it is defined in the first or last meta viewport, at the moment it is defined, the value applies.

````html
	<meta name="viewport" content="width=device-width, user-scalable=yes" />
	<meta name="viewport" content="width=device-width, user-scalable=yes,maximum-scale=1" />
````
The `maximum-scale` is set to 1 in the last viewport
- Android/FF -> The user can't zoom 
- Android CH -> The user can't zoom 
- Android/Internet-Safari/iOS-> The user can zoom

````html
	<meta name="viewport" content="width=device-width, user-scalable=yes,maximum-scale=1" />
	<meta name="viewport" content="width=device-width, user-scalable=yes" />
````

The `maximum-scale` is set to 1 in the first viewport 
- Android/FF -> The user can't zoom 
- Android/CH -> The user **can** zoom 
- Android/Internet-Safari/iOS-> The user can zoom

As the `<title>`, a meta viewport defined in the `<body>` element applied too, and seems to follow the same patten as defined above ... 

````html
<head>    
	<meta name="viewport" content="width=device-width, user-scalable=yes" />
</head>

<body>
	<meta name="viewport" content="width=device-width, user-scalable=no" />
</body>
````
In that case, in FF/Android, the user can't zoom, cause the last meta set the user-scalable to "no" (even if the meta is in the body, not the head).

-----

I didn't find anything in the spec about what to do with multiple meta viewport (or didn't find anything *easily* :-) ), so based on those tests and the multiplicity of behavior, maybe we need to focus on every meta viewport that are present in the head? 


SO! For applicabilty, I would say:  

> The rule applies to every `<meta name="viewport">` elements with a `content` attribute in a document.

Reply:
Nice work, testing these @audreymaniez 

I agree on the applicability:

>The rule applies to every <meta name="viewport"> elements with a content attribute in a document.

And should definitely add notes on `Accessibility Support` about the above found inconsistencies.